src/loader/loader.js

```diff
@@ -334,6 +344,28 @@ export default class Loader {
 		return null;
 	}
 
+	/**
+	 * Show errors in success handler if necessary.
+	 * @param {*} err
+	 */
+	_reportSuccessHandlerError(err) {
+		const config = this._config;
+
+		if (!config.explainResolutions) {
+			return;
+		}
+
+		console.log('---------------------------------------');
+		console.log('Liferay AMD Loader: Uhandled error in success handler');
```

Comment:
```suggestion
		console.log('Liferay AMD Loader: Unhandled error in success handler');
```

Reply:
Removed the log messages and we'll just print the plain error to the console to emulate standard browser's behaviour.